### PR TITLE
Add local storage to forms

### PR DESCRIPTION
**Pull request by AminaBuhari**
Add local storage to file.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -170,14 +171,47 @@ function closePopup() {
...
+    localStorage.clear();
   }
 });
+
+// const email = document.getElementById("form-email");
+//  email.addEventListener('input', (event) => {
+//     const formemail = email.value;
+//     localStorage.setItem("emailkey", formemail);
+// });
+
```
Comment (Cathella) [inline script.js:201]: - [ ] Kindly remove all commented out blocks of code to keep your code clean and readable

File: script.js [hunk 2]
```diff
@@ -170,14 +171,47 @@ function closePopup() {
...
+    message: document.getElementById("text-area").value,
+  }
+  localStorage.setItem("key",JSON.stringify(formData))
+  
   if (email !== email.toLowerCase()) {
     document.getElementById('error-msg').innerText = 'Email Address should be in Lower case';
   } else {
     document.forms[0].submit();
+    localStorage.clear();
   }
```
Comment (Cathella) [inline script.js:193]: - [ ] Form values are not stored in `localStorage` as required by the project milestone. Please find out why and fix the bug.

> You can use the guide below 👇🏻

- The form data for all inputs should be stored in a single object, saved to a single local storage registry.
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```
- use the `localStorage.getItem` and `localStorage.setItem` functions to save and retrieve the form data object to the local storage.

```
localStorage.setItem('contactFormInput', JSON.stringify(formData));

let savedFormData = localStorage.getItem('contactFormInput');
savedFormData = JSON.parse(savedFormData);

```

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @team 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @AminaBuhari && @mrjuda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**